geometry/mesh.cpp

```diff
@@ -119,18 +119,39 @@ void Mesh::skirt( const math::Point3 & down ) {
             // add new faces
             if ( edge.status == Status::FW ) {
 
-                faces.emplace_back( vdownmap[edge.v1], edge.v2, edge.v1,
-                           tdownmap[edge.t1], edge.t2, edge.t1 );
-                faces.emplace_back( vdownmap[edge.v1], vdownmap[edge.v2], edge.v2,
-                           tdownmap[edge.t1], tdownmap[edge.t2], edge.t2 );
+                faces.emplace_back(
+                    static_cast<geometry::Face::index_type>(vdownmap[edge.v1])
+                    , static_cast<geometry::Face::index_type>(edge.v2)
+                    , static_cast<geometry::Face::index_type>(edge.v1)
+                    , static_cast<geometry::Face::index_type>(tdownmap[edge.t1])
+                    , static_cast<geometry::Face::index_type>(edge.t2)
+                    , static_cast<geometry::Face::index_type>(edge.t1));
+                faces.emplace_back(
+                    static_cast<geometry::Face::index_type>(vdownmap[edge.v1])
+                    , static_cast<geometry::Face::index_type>(vdownmap[edge.v2])
+                    , static_cast<geometry::Face::index_type>(edge.v2)
+                    , static_cast<geometry::Face::index_type>(tdownmap[edge.t1])
+                    , static_cast<geometry::Face::index_type>(tdownmap[edge.t2])
+                    , static_cast<geometry::Face::index_type>(edge.t2));
```

Comment:
Wouldn't it be clearer to use the same index type as in the `mesh` class itself? I personally don't see a reason why to convert from `unsigned int` to `size_t` and then back.